tests/components/gltf-model.test.js

```diff
@@ -0,0 +1,58 @@
+/* global assert, process, setup, suite, test */
+var entityFactory = require('../helpers').entityFactory;
+
+var SRC = '/base/tests/assets/box/Box.gltf';
+
+suite.only('gltf-model', function () {
```

Comment:
got an `only` stuck on here (need to lint that, i do this a lot).

I've been following this template for tests now (https://github.com/aframevr/angle/blob/master/templates/component/tests/index.test.js#L6) ... closured variables for `el` and store the `component instance. saves a lot of proxy variables and grabbing the component.